src/main/ipc/react-devtools.ts

```diff
@@ -0,0 +1,30 @@
+import AdmZip from "adm-zip";
+import { ipcMain } from "electron";
+import fetch from "node-fetch";
+import { join } from "path";
+import { CONFIG_PATHS } from "src/util.mjs";
+import { RepluggedIpcChannels } from "../../types";
+
+const OUTPUT_PATH = join(CONFIG_PATHS["react-devtools"]);
+
+// https://github.com/facebook/react/issues/25843#issuecomment-1406766561
+const EXTENSION_URL =
+  "https://github.com/mondaychen/react/raw/017f120369d80a21c0e122106bd7ca1faa48b8ee/packages/react-devtools-extensions/ReactDevTools.zip";
```

Comment:
Per internal discussion, we want to move this to some source we control, either forking this to our GH org or hosting it directly on the server.